webapp/components/MasonryGrid/MasonryGrid.vue

```diff
@@ -27,7 +26,14 @@ export default {
 }
 </script>
 
-<style>
+<style lang="scss">
+/* dirty fix to override broken styleguide inline-styles */
+.ds-grid {
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr)) !important;
+  gap: 16px !important;
+  grid-auto-rows: 20px;
+}
```

Comment:
These values are actually defined in the styleguide but for some reason they get messed up on the way from the styleguide to our repo, so I just copied them in here until we get to migrating the `grid` component.